### PR TITLE
Siri Hemsley - React Native App

### PR DESCRIPTION
**Pull request by sirimatsdotter**
**My reflections on how this week's project turned out:**
This week was really hard for me and I actually started over from scratch this weekend and build a completely different app than I started with. But In the end I'm actually pretty happy with what I achieved given the time I spent!

I believe I have been able to implement both navigation and API request with interaction. 

**Things I'd like to have clarified or explained in more detail:**
I'm super confused about how to apply the flex: 1/or other number on the children of a parent. I could not get it to work well. So at the moment my details page is not looking very good since my textwrapper had to have a set height because otherwise the children would not be covered by the black/transparent background for some reason.....

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,25 +1,66 @@
...
+        />
+        <Stack.Screen name="Upcoming" component={Upcoming} 
+        options={{
+          headerStyle: {
+            backgroundColor: 'black',
+          },
+          headerTintColor: '#fff',
+        }}
+        />
+        <Stack.Screen name="Detail" component={Detail}
```
Comment (KarolCislowski) [inline App.js:52]: I was not able to open the detail page of any of the movies 😭

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please fix the detail page 😸

File: App.js (hunk 1 above)
Comment (sirimatsdotter) [reply App.js:52]: @KarolCislowski - I have now fixed the problem!

**Review by KarolCislowski — APPROVED** [review]
Nice project 😸

**Pull request closed without merging**